tests/app/views/contexts/summary/test_question.py

```diff
@@ -1098,7 +1087,7 @@ def test_get_answer_method(self):
                 question = Question(
                     question_schema,
                     answer_store=answer_store,
-                    schema=self.address_questionnaire_schema(),
+                    schema=self.address_questionnaire_schema("Newline"),
```

Comment:
Instead of this, I would put `self.schema = self.address_questionnaire_schema(concatenation_type)` inside this sub test.

Reply:
Moved it up to top of sub test, I hope that's what you meant?